Firmware/PlatformIO/main_mcu/src/bms.h

```diff
@@ -60,6 +60,8 @@ class BMS
     }
 
     void Tick(std::chrono::milliseconds elapsed_time);
+    
+    void GetStateOfEnergy();
```

Comment:
Since this function doesn't actually return (get) anything, I would probably rename it to CalculateSOE() instead of GetSOE(). You can decide whether to have it be SOE or StateOfEnergy